code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -122,9 +122,17 @@
 	heart_rate_decrease = 1
 	taste_description = "a safe refuge"
 
+/datum/reagent/medicine/cryoxadone/reaction_mob(mob/living/M, method=REAGENT_TOUCH, volume, show_message = 1)
+	if(iscarbon(M))
+		if(method == REAGENT_INGEST && M.bodytemperature < TCRYO)
+			data = "Ingested"
+			if(show_message)
+				to_chat(M, "<span class='warning'>[src] freezes solid in your mouth!</span>") //Burn damage already happens on ingesting
```

Comment:
Might wanna avoid specifically mentioning the mouth as a bunch of non-mouth-based transfers (syringes, hyposprays, IV bags) also use `REAGENT_INGEST`

Reply:
done!